src/main/java/com.marketdata.api/MarketDataController.java

```diff
@@ -10,13 +8,22 @@
 @Controller
 public class MarketDataController {
 
+    /**
+     * index page
+     * @return
+     */
```

Comment:
Added some JavaDoc here to reduce future technical debt.

Reply:
JavaDoc adds value when it describes something that cannot be given from the context.  I'd enhance this a bit.